ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/results/TransactionPendingResult.java

```diff
@@ -77,6 +77,10 @@ public class TransactionPendingResult implements TransactionResult {
   private final String r;
   private final String s;
 
+  private final String blockHash;
+  private final String blockNumber;
+  private final String transactionIndex;
```

Comment:
Would these ever _not_ be null for pending?

Reply:
always null